### PR TITLE
Allowing service destroyer to unbind apps and warn users

### PR DESCRIPTION
**Pull request by ramirezg**
screen shot 2015-08-13 at 10 56 34 am
screen shot 2015-08-13 at 10 56 38 am
screen shot 2015-08-13 at 10 56 44 am
screen shot 2015-08-13 at 11 25 05 am
screen shot 2015-08-13 at 11 25 11 am

### CODE REVIEW

**Comment by ramirezg** [comment]
@jcscottiii this should be done now.

File: static/tests/Ctrl_tests.js [hunk 1]
```diff
@@ -504,9 +537,16 @@ describe('AppCtrl', function() {
     });
 
 
-        it('hould put the available services into the app along with the boundService obj if it\'s bound', function() {
-            expect(scope.availableServices).toEqual([{metadata: { guid: 'serviceguid' }, boundService: { guid: 'serviceguid' }}]);
-        });
+    it('hould put the available services into the app along with the boundService obj if it\'s bound', function() {
```
Comment (jcscottiii) [inline static/tests/Ctrl_tests.js]: hould - > should

**Comment by jcscottiii** [comment]
your unit test for deleteServiceInstance should include a check that it shows the serviceInstances tab after completing successfully

**Comment by jcscottiii** [comment]
Needs to be rebased

**Comment by ramirezg** [comment]
@jcscottiii I've added a note to test the directive where those functions are located.

**Pull request merged**